apps/api_web/lib/api_web/api_controller_helpers.ex

```diff
@@ -179,6 +179,7 @@ defmodule ApiWeb.ApiControllerHelpers do
     split_include =
       case params["include"] do
         nil -> []
+        %{} -> []
         include -> include |> String.split(~r"[.,]") |> MapSet.new()
```

Comment:
What about having this as a list match, and letting everything else fall through?
```elixir
case Map.get(params, "include") do
  [_ | _] -> include |> String.split(~r"[.,]") |> MapSet.new()
  _ -> []
end
```

Also, can you test whether `String.split(include, ~r"[.,]")` or `String.split(include, [",", "."])` is more efficient?